model/fn-execution/src/main/proto/beam_fn_api.proto

```diff
@@ -609,16 +609,20 @@ message Elements {
     // represents the producer of these timers.
     string transform_id = 2;
 
+    // (Optional) The local timer family name used to identify the associated
+    // timer family specification
+    string timer_family_tag = 3;
```

Comment:
`timer_id` == `timer_family_tag` since the `timer_id` represents the `key` from `timer_family_specs`.

Note that we will either need to remove `timer_specs` from ParDoPayload or ensure that `timer_specs` and `timer_family_specs` don't share any keys.

Reply:
We should agree on terminology. The "family" represents the set of timers, within that one has multiple [dynamic] tags. I would call this `timer_family_id`. 

We should not have both `timer_specs` and `timer_family_specs`.